lib/index.js

```diff
@@ -190,44 +46,8 @@ function getLink(obj, linkName, altName) {
   }
 }
 
-function setCookie(name, value, expiresAt) {
-  if (!IS_BROWSER) {
-    return;
-  }
-
-  var expiresText = '';
-  if (expiresAt) {
-    expiresText = ' expires=' + isoToDate(expiresAt).toUTCString() + ';';
-  }
-
-  var cookieText = name + '=' + value + ';' + expiresText;
-  log('Set cookie: ' + cookieText);
-  document.cookie = cookieText;
-
-  return cookieText;
-}
-
-function getCookie(name) {
-  if (!IS_BROWSER) {
-    return;
-  }
-
-  var pattern = new RegExp(name + '=([^;]*)'),
-    matched = document.cookie.match(pattern);
-
-  if (matched) {
-    var cookie = matched[1];
-    log('Got cookie: ', cookie);
-    return cookie;
-  }
-}
-
-function deleteCookie(name) {
-  setCookie(name, '', '1970-01-01T00:00:00Z');
-}
-
 function addStateToken(res, options) {
-  var builtArgs = clone(options) || {};
+  var builtArgs = util.clone(options) || {};
```

Comment:
It might be interesting in the future to just use lodash's per method packages for these types of methods:
https://www.npmjs.com/browse/keyword/lodash-modularized